lib/src/mapbox_map.dart

```diff
@@ -53,7 +53,7 @@ class MapboxMap extends StatefulWidget {
       AnnotationType.line,
       AnnotationType.circle,
     ],
-  })  : assert(annotationOrder.length == 4),
+  })  : assert(annotationOrder.length <= 4),
         assert(annotationConsumeTapEvents.length > 0),
         super(key: key);
 
```

Comment:
Should the doc comment for `final List<AnnotationType> annotationOrder` be changed?